### PR TITLE
Add example to rpm_ostree_pkg

### PR DESCRIPTION
**Pull request by Daskan**
expand examples list with 'until' example

##### SUMMARY
Adding example to docs.

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
rpm_ostree_pkg.py

### CODE REVIEW

File: plugins/modules/rpm_ostree_pkg.py [hunk 1]
```diff
@@ -55,6 +55,17 @@
...
     state: absent
+
+# In case a different transaction is currently running the module would fail.
+# Adding a delay can help mitigate this problem.
+- name: Install overlay package
+  community.general.rpm_ostree_pkg:
+    name: nfs-utils
+    state: present
+  register: rpm_ostree_pkg
+  until: rpm_ostree_pkg.failed is false
```
Comment (felixfontein) [inline plugins/modules/rpm_ostree_pkg.py:66]:
```suggestion
  until: rpm_ostree_pkg is not failed
```

File: plugins/modules/rpm_ostree_pkg.py [hunk 2]
```diff
@@ -55,6 +55,17 @@
   community.general.rpm_ostree_pkg:
     name: nfs-utils
     state: absent
+
+# In case a different transaction is currently running the module would fail.
+# Adding a delay can help mitigate this problem.
```
Comment (felixfontein) [inline plugins/modules/rpm_ostree_pkg.py:60]:
```suggestion
# Adding a delay can help mitigate this problem:
```

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution!

**Comment by russoz** [comment]
Other than the adjustments pointed by @felixfontein , this LGTM

**Comment by felixfontein** [comment]
@Daskan thanks for your contribution!
@russoz thanks for reviewing!

**Pull request merged**